dashboard/src/components/entitypage/elements/VersionControl.js

```diff
@@ -30,10 +30,16 @@ const VersionControl = ({
   handleVersionChange,
   entityPage,
   convertTimestampToDate,
+  local,
 }) => {
   const classes = useStyles();
-  const createdDate =
-    entityPage.resources["versions"][version]["metadata"]["created"];
+  let createdDate;
+  if (local) {
```

Comment:
Same as above, all the logic between implementations should live in a class not across multiple if statements. This makes it easier to add new implementations and encapsulates all the differences next to eachother as opposed to across many different files.